predictions_FB.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from fbprophet import Prophet
+import yfinance as yf
+import math
+import os
+import sys
+import pandas as pd
+pd.options.mode.chained_assignment = None
+
+
+class suppress_stdout_stderr(object):
+    '''
+    Used to supress the verbose output of the inner workings of the FB prophet model
+    '''
+    def __init__(self):
+        self.null_fds = [os.open(os.devnull, os.O_RDWR) for x in range(2)]
+        self.save_fds = (os.dup(1), os.dup(2))
+
+    def __enter__(self):
+        os.dup2(self.null_fds[0], 1)
+        os.dup2(self.null_fds[1], 2)
+
+    def __exit__(self, *_):
+        os.dup2(self.save_fds[0], 1)
+        os.dup2(self.save_fds[1], 2)
+        os.close(self.null_fds[0])
+        os.close(self.null_fds[1])
```

Comment:
Again, we shouldn't be interacting with any `os` methods.

Reply:
The only way to supress the output seems to be this. This is on the official fbprophet repo [issue](https://github.com/facebook/prophet/issues/223)

Reply:
What about [this](https://github.com/facebook/prophet/issues/223#issuecomment-337415032)? This is from the same issue thread. Can you try this out?